lib/rspec/core/did_you_mean/suggestions.rb

```diff
@@ -0,0 +1,55 @@
+module RSpec
+  module Core
+    module DidYouMean
+      # Service object to provide did_you_mean suggestions
+      # based on https://github.com/yuki24/did_you_mean
+      class Suggestions
+        CUT_OFF = 0.834 # Lowest acceptable proximity to be considered probable
+        MAX_SUGGESTIONS = 3 # Maximum number of suggestions that can be provided.
+        attr_reader :relative_file_name, :exception
+
+        def initialize(relative_file_name, exception)
+          @relative_file_name = relative_file_name
+          @exception = exception
+        end
+
+        # provide probable suggestions if a LoadError
+        def call
+          return unless RUBY_VERSION.to_f >= 2.0
+          return unless exception.class == LoadError
```

Comment:
We probably shouldn't call this at all if we don't have the error we expect.

Reply:
Done